documentation/APIs/openapi3-core.yaml

```diff
@@ -98,6 +108,186 @@ paths:
+      parameters:
+        - in: path
+          name: accountId
+          schema:
+            type: string
+          example: "0xf532a53545103276b46876c473846d98648ee418468bce76df4868648dd73e5d"
+          required: true
+          description: Identifier of the account.
+        - in: query
+          name: slotIndex
+          schema:
+            type: integer
+          example: 1622568540
+          required: false
+          description: Check the account status for a specific slot index.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountResponse'
+
+  /api/core/v3/accounts/{accountId}/mana:
+    get:
+      tags:
+        - accounts
+      summary: Returns the potential and stored mana of the account. If slotIndex is not provided, the returned status would be current slot index. 
+      description: >-
+        Returns mana of an account.
+      parameters:
+        - in: path
+          name: accountId
+          schema:
+            type: string
+          example: "0xf532a53545103276b46876c473846d98648ee418468bce76df4868648dd73e5d"
+          required: true
+          description: Identifier of the account.
+        - in: query
+          name: slotIndex
+          schema:
+            type: integer
+          example: 1622568540
+          required: false
+          description: Check the mana of an account for a specific slot index.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountManaResponse'
+
+  /api/core/v3/rewards/{outputId}:
+    get:
+      tags:
+        - rewards
+      summary: Returns the totally available rewards of the output ID. 
+      description: >-
+        Returns mana of an account.
+      parameters:
+        - in: path
+          name: outputId
+          schema:
+            type: string
+          example: "0xf532a53545103276b46876c473846d98648ee418468bce76df4868648dd73e5d"
+          required: true
+          description: Identifier of the output.
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountManaResponse'
+
+  /api/core/v3/committee:
+    get:
+      tags:
+        - committee
+      summary: Return the information of committee members. 
+      description: >-
+        TODO, performance factor, total pool stake
+      responses:
+        '200':
+          description: "Successful operation."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AccountStakingResponse'
+
+  /api/core/v3/staking/{accountId}:
```

Comment:
We don't have this in code yet, afaik, so it's hard to tell what this should return. We will need to keep track of all stakers for the current epoch when we track the activity and calculate their total pool stake at some point. We do this for all registered stakers. Perhaps we can keep track of this until the next time it's updated. Then we could  return the total stake of a pool from this endpoint.